Zilon.Core/Zilon.Core/Persons/MonsterPerson.cs

```diff
@@ -31,9 +31,13 @@ public class MonsterPerson : IPerson
 
         public IMonsterScheme Scheme { get; }
 
+        public PhysicalSize PhysicalSize { get => PhysicalSize.Size1; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a way to break out of this property accessor's recursion.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5310892)